### PR TITLE
Update release notes for 2.1.1

### PR DESCRIPTION
**Pull request by mluu12**
Closes https://github.com/influxdata/docs-v2/issues/3116

Added the CLI release notes for 2.1.1. 

- [ ] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [ ] Tests pass (no build errors)
- [ ] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ menu:
     name: influx CLI 
 ---
 
+## v2.1.1 [unreleased]
```
Comment (danxmoran) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:11]: Everything other than this file seems to be unrelated to the CLI release?

File: content/influxdb/cloud/reference/release-notes/influx-cli.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ menu:
...
+
+## Go Version 
+
+This release upgrades the project to `go` version 1.17.
+
+## Bug Fixes 
+
+- Fix shell completion for top-level `influx` commands.
+- Make global `--http-debug` flag visible in help text.
```
Comment (danxmoran) [inline content/influxdb/cloud/reference/release-notes/influx-cli.md:20]: Do you want/need to add this flag to the detailed reference docs for each CLI command?

Comment (russorat) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:20]: Yes. we should consider adding it here. that can be done in another PR. https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/#flags

File: content/influxdb/cloud/reference/release-notes/influx-cli.md (hunk 1 above)
Comment (mluu12) [reply content/influxdb/cloud/reference/release-notes/influx-cli.md:11]: Hi @danxmoran! Yes I believe I made a mistake! This is strange though because I made this branch immediately before adding edits to the release notes doc and then sent it out immediately after I was done with the work... and when I was staging changes VS code showed me that only that one doc was affected.... But yes, everything in the reference notes doc is only thing related to this PR! Sorry about that!

**Review by danxmoran — COMMENTED** [review]
Everything in the notes LGTM, will defer to somebody from the docs team on whether it's OK to merge with the unrelated changes
